WordPressKit.podspec

```
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'WordPressKit'
-  s.version       = '4.57.1'
+  s.version       = '4.57.1-beta.1'
```

👋 I think we want to use the following version instead

```suggestion
  s.version       = '4.58.0-beta.1'
```

Ref: paNNhX-33-p2

---

@momo-ozawa Yes. Thanks for catching this!
Fixed in 79608708db328f8cbcda7e17763f2290569226db 👍